src/hooks/__tests__/useCurrentUser.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
Let's please keep tests next to the files they're testing so it's easier to navigate and explore what's not tested.